app/(tabs)/index.tsx

```diff
@@ -1,10 +1,104 @@
-import React from 'react';
-import { Text, View } from 'react-native';
+import React, { useState } from 'react';
+import { Animated, Pressable, Text, View } from 'react-native';
 
+import McPosLogo from '../../assets/icon/mcPosLogo.svg';
+import AdminModal from '../../components/AdminModal';
+import CategoryTabs from '../../components/menuSelection/CategoryTabs';
+import MenuGrid from '../../components/menuSelection/MenuGrid';
+import OrderSection from '../../components/order/OrderSection';
+import { useButtonAnimation } from '../../hooks/useButtonAnimation';
+
+/**
+ * 메뉴 선택 메인 화면 컴포넌트
+ * - 카페 메뉴 선택, 주문 관리, 관리자 모드 접근을 담당하는 메인 화면
+ */
 export default function MenuSelection() {
+  // 상태 및 애니메이션 관리
+  const adminAnimation = useButtonAnimation(); // 관리자 버튼 애니메이션
+  const [isAdminModalVisible, setIsAdminModalVisible] = useState(false); // 관리자 모달 표시 상태
+
+  // 이벤트 핸들러
+  /**
+   * 관리자 모드 버튼 클릭 핸들러
+   * 관리자 모달을 열어 비밀번호 입력을 요청
+   */
+  const handleAdminPress = () => {
+    setIsAdminModalVisible(true);
+  };
+
+  /**
+   * 관리자 모달 닫기 핸들러
+   */
+  const closeAdminModal = () => {
+    setIsAdminModalVisible(false);
+  };
+
+  /**
+   * 관리자 비밀번호 확인 핸들러
+   * @param password - 입력된 비밀번호
+   */
+  const handleAdminConfirm = (password: string) => {
+    // 비밀번호 검증 로직
+    if (password === 'admin123') {
```

Comment:
```suggestion
    if (password === MASTER_PASSWORD) {
```
처럼 마스터 비번을 상수로 따로 관리하면 좋을듯. 별도 상수 파일에 관리해도 좋고

Reply:
넵 😃😃